Unable to build Julia from source after `make cleanall && git clean -dfx`

See PR title. I've run these commands on a fresh clone of Julia (f163ed8ce650ce0814afdd5a2d54e39e275aeeb7) on macOS:

- Build command:

```
make cleanall && git clean -dfx && make -j
```

- Output:

```
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/base
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/test
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/Compiler
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/JuliaSyntax
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/JuliaLowering
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
rm -f ./pcre_h.jl
rm -f ./errno_h.jl
rm -f ./build_h.jl
rm -f ./build_h.jl.phony
rm -f ./features_h.jl
rm -f ./uv_constants.jl
rm -f ./file_constants.jl
rm -f ./version_git.jl
rm -f ./version_git.jl.phony
rm -f ./version_git_dirty
rm -f /Users/dnetto/RAI/julia-master/usr/lib/julia/lib*.dylib*
rm -rf _build/* deps/* docbuild.log UnicodeData.txt
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
rm -fr /Users/dnetto/RAI/julia-master/usr/lib/libjulia-internal* /Users/dnetto/RAI/julia-master/usr/lib/libjulia-codegen*
rm -rf /Users/dnetto/RAI/julia-master/usr/lib/libccalltest* /Users/dnetto/RAI/julia-master/usr/lib/libllvmcalltest* /Users/dnetto/RAI/julia-master/usr/lib/libccalllazyfoo* /Users/dnetto/RAI/julia-master/usr/lib/libccalllazybar*
rm -f ./julia_flisp.boot* ./julia_flisp.boot.inc* ./jl_internal_funcs.inc* ./jl_data_globals_defs.inc*
rm -f ./*.dbg.obj ./*.o ./*.dwo ./*.dylib ./*.a ./*.h.gen
rm -f ./julia.expmap*
rm -f ./julia_version.h
rm -f ./compile_commands.json*
rm -f ./*.o ./*.dbg.obj
rm -f /Users/dnetto/RAI/julia-master/usr/bin/julia*
rm -f ./julia.expmap*
rm -f ./Info.plist*
make[2]: /Users/dnetto/RAI/julia-master/usr/bin/julia: Command not found
rm -f /Users/dnetto/RAI/julia-master/test/embedding/embedding-debug /Users/dnetto/RAI/julia-master/test/embedding/embedding
rm -f /Users/dnetto/RAI/julia-master/test/gcext/gcext-debug /Users/dnetto/RAI/julia-master/test/gcext/gcext
rm -f /Users/dnetto/RAI/julia-master/test/gcext/foreignlib.dylib
rm -f /Users/dnetto/RAI/julia-master/test/gcext/foreignlib-debug.dylib
rm -rf .lit_test_times.txt Output
make[2]: /Users/dnetto/RAI/julia-master/usr/bin/julia: Command not found
make[2]: /Users/dnetto/RAI/julia-master/usr/bin/julia: Command not found
rm -f /Users/dnetto/RAI/julia-master/test/trimming/hello /Users/dnetto/RAI/julia-master/test/trimming/trimmability /Users/dnetto/RAI/julia-master/test/trimming/basic_jll /Users/dnetto/RAI/julia-master/test/trimming/hello-o.a /Users/dnetto/RAI/julia-master/test/trimming/trimmability-o.a /Users/dnetto/RAI/julia-master/test/trimming/basic_jll-o.a /Users/dnetto/RAI/julia-master/test/trimming/libsimple-o.a /Users/dnetto/RAI/julia-master/test/trimming/libsimple.dylib /Users/dnetto/RAI/julia-master/test/trimming/capplication /Users/dnetto/RAI/julia-master/test/trimming/bindinginfo_libsimple.json /Users/dnetto/RAI/julia-master/test/trimming/Manifest.toml
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Pkg
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/Pkg
rm -f ./Pkg-1e90f07f9f28e9cec60c5aea0e55302a02164b10/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Statistics
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/Statistics
rm -f ./Statistics-22dee82f9824d6045e87aa4b97e1d64fe6f01d8d/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LazyArtifacts
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/LazyArtifacts
rm -f ./LazyArtifacts-e4cfc39598c238f75bdfdbdb3f82c9329a5af59c/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibCURL
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/LibCURL
rm -f ./LibCURL-9ea5c5d6f5b88615d9fe23379b7f951787b99fd3/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/DelimitedFiles
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/DelimitedFiles
rm -f ./DelimitedFiles-aac8c59e58cbf961fa15baf4d866901d9d1e6980/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Downloads
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/Downloads
rm -f ./Downloads-4e20d029c723199c0b8ea0e2418ff240d25ddaef/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/ArgTools
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/ArgTools
rm -f ./ArgTools-89d19599208c02bfa9609d4578ab72eabe6e8eee/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Tar
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/Tar
rm -f ./Tar-d236fa0affb2ab90c78798b01bb1d64615785354/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/NetworkOptions
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/NetworkOptions
rm -f ./NetworkOptions-7034c55dbf52ee959cabd63bcbe656df658f5bda/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SuiteSparse
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/SuiteSparse
rm -f ./SuiteSparse-e8285dd13a6d5b5cf52d8124793fc4d622d07554/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SparseArrays
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/SparseArrays
rm -f ./SparseArrays-26c80c8b45dc2dca92788332a40a99b6c360d05a/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/StyledStrings
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/StyledStrings
rm -f ./StyledStrings-9bb8ffdd8c2858cced7b6b6fcee85be41c9a1867/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SHA
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/SHA
rm -f ./SHA-876bc0400f9a457eb2736388fc3d0fbe9460fc7d/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Distributed
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/Distributed
rm -f ./Distributed-cd9219573d736b036077dff3cadddf369516d495/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/JuliaSyntaxHighlighting
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/JuliaSyntaxHighlighting
rm -f ./JuliaSyntaxHighlighting-84fb1fd08824736de14aaa94265df756474e0bdf/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LinearAlgebra
rm -f /Users/dnetto/RAI/julia-master/usr/manifest/v1.14/LinearAlgebra
rm -f ./LinearAlgebra-b599095ef3da7ba7e950ee4700a3ba0fea047949/build-compiled
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Artifacts
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Base64
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/CRC32c
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Dates
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/FileWatching
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Future
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/InteractiveUtils
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Libdl
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibGit2
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Logging
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Markdown
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Mmap
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Printf
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Profile
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Random
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/REPL
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Serialization
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SharedArrays
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Sockets
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Test
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/TOML
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Unicode
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/UUIDs
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/MozillaCACerts_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/dSFMT_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/GMP_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibCURL_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibGit2_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/libLLVM_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibSSH2_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibUV_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenSSL_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/MPFR_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/nghttp2_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/libblastrampoline_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenBLAS_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenLibm_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/p7zip_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/PCRE2_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SuiteSparse_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Zlib_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Zstd_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LLVMLibUnwind_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/CompilerSupportLibraries_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibUnwind_jll
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LLD_jll
for module in Pkg Statistics LazyArtifacts LibCURL DelimitedFiles Downloads ArgTools Tar NetworkOptions SuiteSparse SparseArrays StyledStrings SHA Distributed JuliaSyntaxHighlighting LinearAlgebra ; do \
                rm -rf /Users/dnetto/RAI/julia-master/stdlib/${module}-*; \
        done
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Artifacts/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/FileWatching/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Libdl/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SHA/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/libblastrampoline_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenBLAS_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Random/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LinearAlgebra/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Sockets/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/ArgTools/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Base64/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/CRC32c/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Dates/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/DelimitedFiles/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Distributed/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Downloads/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Future/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/InteractiveUtils/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/JuliaSyntaxHighlighting/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LazyArtifacts/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibGit2/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibCURL/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Logging/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Markdown/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Mmap/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/NetworkOptions/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Profile/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Printf/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Pkg/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/REPL/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Serialization/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SharedArrays/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SparseArrays/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Statistics/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/StyledStrings/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SuiteSparse_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/SuiteSparse/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Tar/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Test/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/TOML/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Unicode/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/UUIDs/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/dSFMT_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/GMP_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/libLLVM_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LLD_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LLVMLibUnwind_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibUnwind_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibUV_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibCURL_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibSSH2_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/LibGit2_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/nghttp2_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/MozillaCACerts_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/MPFR_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenLibm_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/OpenSSL_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/PCRE2_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/p7zip_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Zlib_jll/src: No such file or directory
find: /Users/dnetto/RAI/julia-master/usr/share/julia/stdlib/v1.14/Zstd_jll/src: No such file or directory
rm -rf /Users/dnetto/RAI/julia-master/usr/share/julia/compiled
rm -f ./stdlib/*.image
rm -f /Users/dnetto/RAI/julia-master/julia
rm -f /Users/dnetto/RAI/julia-master/*.tar.gz
rm -f /Users/dnetto/RAI/julia-master/usr/tools/stringreplace \
           /Users/dnetto/RAI/julia-master/light-source-dist.tmp /Users/dnetto/RAI/julia-master/light-source-dist.tmp1 \
           /Users/dnetto/RAI/julia-master/full-source-dist.tmp /Users/dnetto/RAI/julia-master/full-source-dist.tmp1
rm -fr /Users/dnetto/RAI/julia-master/usr/lib/julia
rm -fr /Users/dnetto/RAI/julia-master/usr/share/julia/extras
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
make[1]: /Users/dnetto/RAI/julia-master/usr/tools/llvm-config: Command not found
/Library/Developer/CommandLineTools/usr/bin/make -C /Users/dnetto/RAI/julia-master/src/flisp clean BUILDDIR='/Users/dnetto/RAI/julia-master/src/flisp'
rm -f /Users/dnetto/RAI/julia-master/src/flisp/*.o
rm -f /Users/dnetto/RAI/julia-master/src/flisp/*.dbg.obj
rm -f /Users/dnetto/RAI/julia-master/src/flisp/*.a
rm -f /Users/dnetto/RAI/julia-master/src/flisp/flisp
rm -f /Users/dnetto/RAI/julia-master/src/flisp/flisp-debug
rm -f /Users/dnetto/RAI/julia-master/src/flisp/compile_commands.json*
rm -f /Users/dnetto/RAI/julia-master/src/flisp/host/*
/Library/Developer/CommandLineTools/usr/bin/make -C /Users/dnetto/RAI/julia-master/src/support clean BUILDDIR='/Users/dnetto/RAI/julia-master/src/support'
rm -f /Users/dnetto/RAI/julia-master/src/support/*.o
rm -f /Users/dnetto/RAI/julia-master/src/support/*.dbg.obj
rm -f /Users/dnetto/RAI/julia-master/src/support/*.a
rm -f /Users/dnetto/RAI/julia-master/src/support/*~ *#
rm -f /Users/dnetto/RAI/julia-master/src/support/core*
rm -f /Users/dnetto/RAI/julia-master/src/support/libsupport.a
rm -f /Users/dnetto/RAI/julia-master/src/support/libsupport-debug.a
rm -f /Users/dnetto/RAI/julia-master/src/support/compile_commands.json*
rm -f /Users/dnetto/RAI/julia-master/src/support/host/*
make[1]: Nothing to be done for `clean-libuv'.
rm -f scratch/utf8proc-53f0bac76f5d5bd3ed69d1a06ee5b200c8940610/build-compiled
/Library/Developer/CommandLineTools/usr/bin/make -C scratch/utf8proc-53f0bac76f5d5bd3ed69d1a06ee5b200c8940610 clean
make: *** scratch/utf8proc-53f0bac76f5d5bd3ed69d1a06ee5b200c8940610: No such file or directory.  Stop.
make[1]: [clean-utf8proc] Error 2 (ignored)
rm -fr /Users/dnetto/RAI/julia-master/usr /Users/dnetto/RAI/julia-master/usr-staging
Creating usr/etc/julia/startup.jl
Copying in usr/share/man/man1/julia.1
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/julia-config.jl /Users/dnetto/RAI/julia-master/usr/share/julia/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/juliac.jl /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/abi_export.jl /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/juliac-buildscript.jl /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
mkdir -p /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/juliac-trim-stdlib.jl /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/juliac-trim-base.jl /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
/Users/dnetto/RAI/julia-master/contrib/install.sh 755 /Users/dnetto/RAI/julia-master/contrib/juliac/Artifacts.toml /Users/dnetto/RAI/julia-master/usr/share/julia/juliac/
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   491  100   491    0     0    581      0 --:--:-- --:--:-- --:--:--   581
100   461  100   461    0     0    492      0 --:--:-- --:--:-- --:--:--   491
100   475  100   475    0     0    463      0  0:00:01  0:00:01 --:--:--   463
100   483  100   483    0     0    439      0  0:00:01  0:00:01 --:--:--   439
0    0      0    0    0    0      0   0   0     0     0        0    0        0    0   - - :0- --:--:-- - :0-:-0 0 :00:10 0-:-0:1- --   0     0     00     0        0  0        00        0  0        0    0        0    0   - - :0- --:--:-- - :0-:-0 0 :00:10 0-:-0:1-       0  0      0    0    0  0         00    0  0          0   0   0  0      0      0    0             0 0      0            00  0--100   477  100   477    0     0    420      0  0:00:01  0:00:01 --:--:--   4200 1   - - :00--:--     0
     0    0      0  0    0     0     0      0  0        0  0          0  0          0  0- --:--:--:--:-- -  0 :000:01: 0-0-::0-1- :-    00          00        00       0    0      0   0        0      0  0     0      0  -  - : -0- :----: - -0::-0-0 : 001: 0-0-::0-1-100   439  100   439    0     0    370      0  0:00:01  0:00:01 --:--:--   370
    00          00        00          00        00          0      0 0          0 0 --: - - : - -  0  0-:-0:0-:-0:1- -- - :0-:-0:0-:    00        0     0    0      00      0        00        0    0      0      0   - -0: - - : - -  0  0-:-0:-0-::0-1 ---: - -0::-0-0100  464k  100  464k    0     0   395k      0  0:00:01  0:00:01 --:--:--  395k
100 1844k  100 1844k    0     0  1521k      0  0:00:01  0:00:01 --:--:-- 1521k
100   541  100   541    0     0    428      0  0:00:01  0:00:01 --:--:--   428
100   481  100   481    0     0    392      0  0:00:01  0:00:01 --:--:--   392
100   483  100   483    0     0    390      0  0:00:01  0:00:01 --:--:--   391
100   475  100   475    0     0    382      0  0:00:01  0:00:01 --:--:--   383
100  482k  100  482k    0     0   369k      0  0:00:01  0:00:01 --:--:--  369k
100   507  100   507    0     0    400      0  0:00:01  0:00:01 --:--:--   400
100   481  100   481    0     0    391      0  0:00:01  0:00:01 --:--:--   391
100   495  100   495    0     0    393      0  0:00:01  0:00:01 --:--:--   393
100   483  100   483    0     0    391      0  0:00:01  0:00:01 --:--:--   391
100   483  100   483    0     0    388      0  0:00:01  0:00:01 --:--:--   388
100  374k  100  374k    0     0   280k      0  0:00:01  0:00:01 --:--:--  280k
100   473  100   473    0     0    381      0  0:00:01  0:00:01 --:--:--   382
0100   481  100   481    0     0    382      0  0:00:01  0:00:01 --:--:--   382
83  100   483    0     0    387      0  0:00:01  0:00:01 --:--:--   387
     0    411      0  0:00:01  0:00:01 --:--:--   411
100  470k  100  470k    0     0   346k      0  0:00:01  0:00:01 --:--:--  346k
100   477  100   477    0     0    378      0  0:00:01  0:00:01 --:--:--   378
100   503  100   503    0     0    398      0  0:00:01  0:00:01 --:--:--   398
100   511  100   511    0     0    399      0  0:00:01  0:00:01 --:--:--   400
100  219k  100  219k    0     0   152k      0  0:00:01  0:00:01 --:--:--  152k
100   479  100   479    0     0    364      0  0:00:01  0:00:01 --:--:--   364
100 61074  100 61074    0     0  44129      0  0:00:01  0:00:01 --:--:-- 44129
100   477  100   477    0     0    357      0  0:00:01  0:00:01 --:--:--   357
100   475  100   475    0     0    357      0  0:00:01  0:00:01 --:--:--   357
100  6836  100  6836    0     0   5000      0  0:00:01  0:00:01 --:--:--  5000
100  305k  100  305k    0     0   210k      0  0:00:01  0:00:01 --:--:-- 29.7M
100  152k  100  152k    0     0   106k      0  0:00:01  0:00:01 --:--:-- 3613k
100  505k  100  505k    0     0   341k      0  0:00:01  0:00:01 --:--:-- 9588k
100  240k  100  240k    0     0   164k      0  0:00:01  0:00:01 --:--:-- 4086k
100  436k  100  436k    0     0   294k      0  0:00:01  0:00:01 --:--:-- 4723k
100  819k  100  819k    0     0   532k      0  0:00:01  0:00:01 --:--:-- 6984k
100  141k  100  141k    0     0  95150      0  0:00:01  0:00:01 --:--:--  856k
100 1834k  100 1834k    0     0  1091k      0  0:00:01  0:00:01 --:--:-- 9789k
100  616k  100  616k    0     0   391k      0  0:00:01  0:00:01 --:--:-- 6000k
100  782k  100  782k    0     0   487k      0  0:00:01  0:00:01 --:--:-- 3275k
100 4784k  100 4784k    0     0  2795k      0  0:00:01  0:00:01 --:--:-- 18.5M
100   477  100   477    0     0    275      0  0:00:01  0:00:01 --:--:--   275
100  772k  100  772k    0     0   449k      0  0:00:01  0:00:01 --:--:-- 2662k
100 3593k  100 3593k    0     0  1931k      0  0:00:01  0:00:01 --:--:-- 8947k
100  7691  100  7691    0     0   4251      0  0:00:01  0:00:01 --:--:-- 18712
100  197k  100  197k    0     0  99585      0  0:00:02  0:00:02 --:--:-- 99585
100 2632k  100 2632k    0     0  1196k      0  0:00:02  0:00:02 --:--:-- 3393k
100 13.6M  100 13.6M    0     0  5707k      0  0:00:02  0:00:02 --:--:-- 17.2M
/Library/Developer/CommandLineTools/usr/bin/install_name_tool: warning: changes being made to the file will invalidate the code signature in: /Users/dnetto/RAI/julia-master/usr/libexec/julia/zstd
libwhich.c:185:23: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
struct vector_t dllist()
                      ^
                       void
1 warning generated.
100 64.0M  100 64.0M    0     0  7579k      0  0:00:08  0:00:08 --:--:-- 10.9M
make[1]: *** No rule to make target `/Users/dnetto/RAI/julia-master/src/julia.expmap.in', needed by `julia.expmap'.  Stop.
make[1]: *** Waiting for unfinished jobs....
    CC usr/lib/libllvmcalltest.dylib
    CC usr/lib/libccalltest.dylib
    CC usr/lib/libccalllazyfoo.dylib
make[1]: *** No rule to make target `/Users/dnetto/RAI/julia-master/cli/julia.expmap.in', needed by `julia.expmap'.  Stop.
make[1]: *** Waiting for unfinished jobs....
    CC cli/loader_lib.o
    CC cli/loader_exe.o
    CC cli/loader_symbol_probe.o
    CC cli/loader_library_probe.o
    CC cli/loader_trampolines.o
make: *** [julia-cli-release] Error 2
make: *** Waiting for unfinished jobs....
/bin/sh: line 1: 41316 Abort trap: 6           /Users/dnetto/RAI/julia-master/usr/tools/libwhich -p libcrypto.dylib 2> /dev/null
/bin/sh: line 1: 41338 Abort trap: 6           /Users/dnetto/RAI/julia-master/usr/tools/libwhich -p libssl.dylib 2> /dev/null
    PERL base/pcre_h.jl
    PERL base/errno_h.jl
    PERL base/file_constants.jl
    PERL base/uv_constants.jl
    PERL base/features_h.jl
    PERL base/features_h.jl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0    PERL base/features_h.jl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   425  100   425    0     0   1774      0 --:--:-- --:--:-- --:--:--  1770
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0    PERL base/build_h.jl.phony
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   425  100   425    0     0   1781      0 --:--:-- --:--:-- --:--:--  1778
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6757  100  6757    0     0  17696      0 --:--:-- --:--:-- --:--:-- 17696
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   425  100   425    0     0   1331      0 --:--:-- --:--:-- --:--:--  1328
100  6985  100  6985    0     0  20307      0 --:--:-- --:--:-- --:--:-- 20307
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 12297  100 12297    0     0  29584      0 --:--:-- --:--:-- --:--:-- 29584
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   425  100   425    0     0    970      0 --:--:-- --:--:-- --:--:--   972
100   425  100   425    0     0    879      0 --:--:-- --:--:-- --:--:--   879
100   425  100   425    0     0   1201      0 --:--:-- --:--:-- --:--:--  1203
100   425  100   425    0     0    902      0 --:--:-- --:--:-- --:--:--   904
100   425  100   425    0     0   1085      0 --:--:-- --:--:-- --:--:--  1086
100  6947  100  6947    0     0  11843      0 --:--:-- --:--:-- --:--:-- 11843
100   425  100   425    0     0   1152      0 --:--:-- --:--:-- --:--:--  1154
100 27218  100 27218    0     0  48351      0 --:--:-- --:--:-- --:--:-- 48351
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
 1 0 0       4 2 5     1 0 0      D4l2o5a d     0     0U   1082      0 --:--:-- --:--:-- --:--:--  1084
pload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   425  100   425    0     0    992      0 --:--:-- --:--:-- --:--:--   992
100   425  100   425    0     0    798      0 --:--:-- --:--:-- --:--:--   798
100  6550  100  6550    0     0  13072      0 --:--:-- --:--:-- --:--:-- 13072
    PERL base/version_git.jl.phony
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6985  100  6985    0     0  11987      0 --:--:-- --:--:-- --:--:-- 11987
100  6662  100  6662    0     0  13387      0 --:--:-- --:--:-- --:--:-- 13387
100   425  100   425    0     0   1279      0 --:--:-- --:--:-- --:--:--  1280
100   425  100   425    0     0   1070      0 --:--:-- --:--:-- --:--:--  1073
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 19878  100 19878    0     0  41386      0 --:--:-- --:--:-- --:--:-- 41386
100  6985  100  6985    0     0  13125      0 --:--:-- --:--:-- --:--:-- 13125
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7935  100  7935    0     0  14950      0 --:--:-- --:--:-- --:--:-- 14950
100  6776  100  6776    0     0  10410      0 --:--:-- --:--:-- --:--:-- 10410
100  6406  100  6406    0     0  14961      0 --:--:-- --:--:-- --:--:-- 14961
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7042  100  7042    0     0  13559      0 --:--:-- --:--:-- --:--:-- 13559
100   425  100   425    0     0   1242      0 --:--:-- --:--:-- --:--:--  1242
100   425  100   425    0     0   1351      0 --:--:-- --:--:-- --:--:--  1349
100   425  100   425    0     0   1111      0 --:--:-- --:--:-- --:--:--  1109
100   425  100   425    0     0   1357      0 --:--:-- --:--:-- --:--:--  1353
100   425  100   425    0     0   1302      0 --:--:-- --:--:-- --:--:--  1299
100  6662  100  6662    0     0  16582      0 --:--:-- --:--:-- --:--:-- 16582
100   425  100   425    0     0   1309      0 --:--:-- --:--:-- --:--:--  1307
100   425  100   425    0     0   1191      0 --:--:-- --:--:-- --:--:--  1190
100  6946  100  6946    0     0  14998      0 --:--:-- --:--:-- --:--:-- 14998
100   425  100   425    0     0   1272      0 --:--:-- --:--:-- --:--:--  1272
100  6833  100  6833    0     0  14055      0 --:--:-- --:--:-- --:--:-- 14055
100  6280  100  6280    0     0  15092      0 --:--:-- --:--:-- --:--:-- 15092
100  6731  100  6731    0     0  15267      0 --:--:-- --:--:-- --:--:-- 15267
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5701  100  5701    0     0  12947      0 --:--:-- --:--:-- --:--:-- 12947
100 24138  100 24138    0     0  50839      0 --:--:-- --:--:-- --:--:-- 50839
100 25978  100 25978    0     0  56470      0 --:--:-- --:--:-- --:--:-- 56470
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   477  100   477    0     0   1609      0 --:--:-- --:--:-- --:--:--  1611
100   477  100   477    0     0   1750      0 --:--:-- --:--:-- --:--:--  1753
100   477  100   477    0     0   1656      0 --:--:-- --:--:-- --:--:--  1662
100 13258  100 13258    0     0  32175      0 --:--:-- --:--:-- --:--:-- 32175
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   477  100   477    0     0   1834      0 --:--:-- --:--:-- --:--:--  1841
100 30126  100 30126    0     0  75584      0 --:--:-- --:--:-- --:--:-- 75584
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 41480  100 41480    0     0  91910      0 --:--:-- --:--:-- --:--:-- 91910
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2762  100  2762    0     0   7225      0 --:--:-- --:--:-- --:--:--  7225
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   477  100   477    0     0   1479      0 --:--:-- --:--:-- --:--:--  1481
100   477  100   477    0     0   1563      0 --:--:-- --:--:-- --:--:--  1563
100 15515  100 15515    0     0  37399      0 --:--:-- --:--:-- --:--:--  360k
100 28017  100 28017    0     0  64696      0 --:--:-- --:--:-- --:--:-- 64696
100   477  100   477    0     0   1171      0 --:--:-- --:--:-- --:--:--  1171
    CC src/support/hashing.o
100   477  100   477    0     0    915      0 --:--:-- --:--:-- --:--:--   915
100   477  100   477    0     0   1050      0 --:--:-- --:--:-- --:--:--  1048
100   477  100   477    0     0   1153      0 --:--:-- --:--:-- --:--:--  1152
    CC src/support/timefuncs.o
    CC src/support/utf8.o
    CC src/support/htable.o
    CC src/support/bitvector.o
    CC src/support/ptrhash.o
100  4710  100  4710    0     0   9468      0 --:--:-- --:--:-- --:--:--  9468
    CC src/support/int2str.o
    CC src/support/operators.o
    CC src/support/libsupportinit.o
    CC src/support/strtod.o
110000      447777   100   477    0     0   1007  100       4 707  - - : -0- : - -   -0- : - -1:0-8- --:8      0 --:--:-- --:--:-- - 100    1087477   10     0 --:--:-- --:-0- :  -- --:--:--  1086
477    0     0   1006      0 --:--:-- --:--:-- --:--:--  1006
100   477  100   477    0     0   1033      0 --:--:-- --:--:-- --:--:--  1032
    CC src/support/arraylist.o
    CC src/support/ios.o
    CC src/support/rle.o
100   477  100   477    0     0    992      0 --:--:-- --:--:-- --:--:--   991
100   477  100   477    0     0   1128      0 --:--:-- --:--:-- --:--:--  1127
100   477  100   477    0     0   1167      0 --:--:-- --:--:-- --:--:--  1166
100  251k  100  251k    0     0   425k      0 --:--:-- --:--:-- --:--:--  425k
100  452k  100  452k    0     0   703k      0 --:--:-- --:--:-- --:--:--  703k
100 22610  100 22610    0     0  31092      0 --:--:-- --:--:-- --:--:-- 31092
100 71518  100 71518    0     0   123k      0 --:--:-- --:--:-- --:--:--  123k
100 10500  100 10500    0     0  18217      0 --:--:-- --:--:-- --:--:-- 18217
100  9095  100  9095    0     0  16947      0 --:--:-- --:--:-- --:--:-- 16947
100 30199  100 30199    0     0  48364      0 --:--:-- --:--:-- --:--:-- 48364
100  696k  100  696k    0     0   957k      0 --:--:-- --:--:-- --:--:--  957k
100 48939  100 48939    0     0  82696      0 --:--:-- --:--:-- --:--:-- 82696
    LINK src/support/libsupport.a
    CC usr/lib/libccalllazybar.dylib
make: *** [julia_flisp.boot.inc.phony] Error 2
```

--- a/cli/Makefile
+++ b/cli/Makefile
@@ -176,7 +176,7 @@ $(BUILDDIR)/julia.expmap: $(SRCDIR)/julia.expmap.in $(JULIAHOME)/VERSION
 clean: | $(CLEAN_TARGETS)
 	rm -f $(BUILDDIR)/*.o $(BUILDDIR)/*.dbg.obj
 	rm -f $(build_bindir)/julia*
-	rm -f $(BUILDDIR)/julia.expmap*
+	rm -f $(BUILDDIR)/julia.expmap
 	rm -f $(BUILDDIR)/Info.plist*
 
 .PHONY: clean release debug julia-release julia-debug

--- a/src/Makefile
+++ b/src/Makefile
@@ -588,7 +588,7 @@ clean:
 	-rm -rf $(build_shlibdir)/libccalltest* $(build_shlibdir)/libllvmcalltest* $(build_shlibdir)/libccalllazyfoo* $(build_shlibdir)/libccalllazybar*
 	-rm -f $(BUILDDIR)/julia_flisp.boot* $(BUILDDIR)/julia_flisp.boot.inc* $(BUILDDIR)/jl_internal_funcs.inc* $(BUILDDIR)/jl_data_globals_defs.inc*
 	-rm -f $(BUILDDIR)/*.dbg.obj $(BUILDDIR)/*.o $(BUILDDIR)/*.dwo $(BUILDDIR)/*.$(SHLIB_EXT) $(BUILDDIR)/*.a $(BUILDDIR)/*.h.gen
-	-rm -f $(BUILDDIR)/julia.expmap*
+	-rm -f $(BUILDDIR)/julia.expmap
 	-rm -f $(BUILDDIR)/julia_version.h
 	-rm -f $(BUILDDIR)/compile_commands.json*
 

